examples/Cargo.toml

```diff
@@ -7,6 +7,9 @@ edition = "2018"
 [dependencies]
 chrono = "0.4.19"
 once_cell = "1.5"
+glium = "0.30.1"
+epoxy = "0.1.0"
+libloading = "0.7.0"
```

Comment:
Maybe these can be made optional only for this example so you don't need to build glium just to run the hello world example

Reply:
I'm not sure you can specify dependencies per binary? The alternative is with a non-default feature.

Reply:
I usually use non-default features for that. You can specify the required features per binary so the user would get a message telling what needs to be enabled.